mathclass.js

```diff
@@ -0,0 +1,21 @@
+class mathalg {
+  constructor(arr) {
+    this.arr = arr;
+  }
+
+  sum() {
+    let total_sum = 0;
```

Comment:
Naming of variables

- [ ] variable names should follow Camel Casing
`Bad: total_sum
Good: totalSum`

- [ ] Variable names should be meaningfully and complete. Abbreviations should be avoided
`Bad: xx,xyz, a,cusOrHIs
Good: customerOrderHistory`

- [ ] Do not use underscores (_) for local variable names.
- [ ] All global variables in a class should be prefixed with an underscore (_)so that they can be identified from other local variables
`Bad: globalCustimerName
God: _customerName`

Reply:
This is not Feedback